dp/cloud/python/magma/test_runner/tests/test_dp_with_orc8r.py

```diff
@@ -492,7 +492,7 @@ def then_state_is_eventually(self, expected, request):
     @retry(stop_max_attempt_number=30, wait_fixed=1000)
     def then_logs_are(self, filters: Dict[str, Any], expected: List[str], network: Optional[str] = None):
         actual = self._get_log_types(filters, network)
-        self.assertListEqual(expected, actual)
+        self.assertCountEqual(expected, actual)
```

Comment:
Maybe we should check if sets are equal if we don't care about the order, checking only count may be wrong for example in case there are no filters

Reply:
the name of the method is misleading, it actually compares the content of the containers regardless of the order